test/e2e/tests/phishing-controller/mocks.js

```diff
@@ -158,10 +167,8 @@ function resolveProviderConfigName(providerName) {
   switch (providerName.toLowerCase()) {
     case BlockProvider.MetaMask:
       return 'eth_phishing_detect_config';
-    case BlockProvider.PhishFort:
-      return 'phishfort_hotlist';
     default:
-      throw new Error('Provider name must either be metamask or phishfort');
+      throw new Error('Provider name must be metamask');
```

Comment:
does it still worth it using a switch case here ? we can just replace with an `if` check ?